ansible/roles/es-mapping/files/mappings/userv3-mapping.json

```diff
@@ -889,6 +889,937 @@
           }
         }
       }
+    },
+    "profileDetails": {
```

Comment:
Please take a look for the below and change accordingly
 "userType": {
      "type": "text",
      "fields": {
        "raw": {
          "type": "text",
          "analyzer": "keylower"
        }
      },
      "analyzer": "cs_search_analyzer",
      "search_analyzer": "cs_search_analyzer"
    },
    
    
    remove copy_to and fielddata= true as its not required